docker/images/nginx/Dockerfile

```diff
@@ -1,3 +1,16 @@
 FROM nginx
+
+# Creating self signed certificate for localhost
+RUN mkdir /etc/nginx/ssl
+
+RUN openssl req \
+        -x509 \
+        -nodes \
+        -newkey rsa:2048 \
+        -keyout /etc/nginx/ssl/server.key \
+        -out /etc/nginx/ssl/server.crt \
+        -days 1000 \
+        -subj "/CN=127.0.0.1"
```

Comment:
Also, the generated cert is only valid for `127.0.0.1`; the `subj` should be `/CN=localhost`.